internal/cmd/db_wakeup.go

```diff
@@ -15,8 +15,8 @@ func init() {
 }
 
 var wakeUpDbCmd = &cobra.Command{
-	Use:               "wakeup <db-name>",
```

Comment:
Please keep the old command around as well, preferably hidden. You can use an alias to achieve that

Reply:
Good idea, I have updated it.